dp_branding/index.php

```diff
@@ -36,121 +36,202 @@
 
 echo "<h3 id='color_logo'>" . _("Color Logo") . "</h3>";
 
-echo "<p><b>" . _("SVG file") . ":</b> <a href='dp-logo.svg'>dp-logo.svg</a></p>";
-
-echo "<p><b>" . _("PNG files") . ":</b></p>";
+echo "<h4>" . _("SVG files") . "</h4>";
+
+echo <<<SVGLOGO
+        <table class='branding'>
+        <tr>
+            <td>Transparent<br>Background, light themes</td>
+            <td style='background-color:white;'>
+                <img src='dp-logo.svg'>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <a href='dp-logo.svg'>dp-logo.svg</a>
+            </td>
+        </tr>
+        <tr>
+            <td>Transparent<br>Background, dark themes</td>
+        <td style='background-color:black;'>
+                <img src='dp-logo-dark.svg'>
```

Comment:
instead of linking to the branding/dp-logo-dark.svg, does it make sense to link to the graphics one and only have 1 copy of the file until we need to diff for the pgdp production branch?

Reply:
Same for dp-logo.svg now since it is in master brnach.

Reply:
Since the logo and mark are specific to DP, and we actually would (I assume) prefer those using the code to have their own logo and mark, I would have thought that it should be the other way around, and have the logo and mark only in the `pgdp-production` branch, but that is beyond the scope of this MR and would involve changes in both `master` and `pgdp-production`. However, I'll defer to @cpeel on what the best way to do it is.

Reply:
Some diffs can be ignored by setting the diff view to ignore white space.

Reply:
In this case I would leave the duplicate files. My rationale is that the `dp-logo.svg` file in PROD has existed at https://www.pgdp.net/dp_branding/dp-logo.svg since its inception and we might inadvertently break something external to the site were we to remove it. I also see value in keeping the branding page separate from the location of files in the main codebase since they could change independently (it would be easy to change code in `master` without realizing it would impact `pgdp-production`.